### DISCUSSION TITLE
Barra de progresso ou alguma animação de trabalho

### DISCUSSION THREAD

**Discussion by JefPresotto** [Stirling-Tools/Stirling-PDF#127] [Ideas]
É uma das melhores aplicações que ja utilizei. Parabéns a todos os envolvidos!

Gostaria de deixar como sugestão uma opção que muitos usuarios estão solicitando, a inclusão de alguma barra de progresso ou de trabalho na pagina web, para mostrar que  o arquivo esta em processamento, pois ao executar o processamento em um arquivo PDF, a tela fica parada até que o arquivo apareça processado, então é difícil para o usuario saber se o servidor esta de fato processando o arquivo. 

Mais uma vez, agradeço ao grande esforço de todos.

**Comment by Frooodle** [comment]
I have ticket for this
https://github.com/Frooodle/Stirling-PDF/issues/74

I plan to try add this soon, I agree it's a needed feature! 

And thanks for your comments
